app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -91,21 +92,20 @@ protected void onCreate(Bundle savedInstanceState) {
         prepareListData();
         listAdapter = new NavDrawerListAdapter(this, listDataHeader, listDataChild);
         expListView.setAdapter(listAdapter);
-        expListView.setOnGroupExpandListener(new ExpandableListView.OnGroupExpandListener() {
+        expListView.setOnGroupClickListener(new ExpandableListView.OnGroupClickListener() {
             @Override
-            public void onGroupExpand(int groupPosition) {
-                switch (groupPosition)
-                {
+            public boolean onGroupClick(ExpandableListView parent, View v, int groupPosition, long id) {
+                switch (groupPosition) {
                     case 0:
                         //Swapping ContactPostStaff into the fragment container dynamically
                         Fragment contactPostStaffFragment = new ContactPostStaff();
-                        MainActivity.swapFragmentIn(MainActivity.this,contactPostStaffFragment,ContactPostStaff.TAG,true);
+                        MainActivity.swapFragmentIn(MainActivity.this, contactPostStaffFragment, ContactPostStaff.TAG, true);
                         mDrawer.closeDrawer(GravityCompat.START);
```

Comment:
This is not required to be committed. There are other places on the file that require a space. We can refactor the code later. Please remove this and other similar commits from this PR.